### PR TITLE
Smaller grid

### PR DESCRIPTION
**Pull request by caira044**
Added a smaller grid to the right of the main board.

### CODE REVIEW

File: src/game.css [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+.block-display {
+    height: 300px; 
+    position: relative;
+    
+   
+}
+.grid {
     display: block;
-    border: 1px solid #000000; 
+    border: 4px solid #000000;
```
Comment (rickaw) [inline src/game.css:17]: Was this intentional?

File: src/game.js [hunk 2]
```diff
@@ -28,3 +28,24 @@ fillBox(1, 2)
...
+    line.lineTo(i,800);
+    line.stroke();
+    console.log(i)
+}
+for(let i = 40; i<400; i+=40) {
+    line.moveTo(0,i);
+    line.lineTo(200,i);
+    line.stroke();
+    console.log(i)
+}
```
Comment (rickaw) [inline src/game.js:51]: should we make this 4x4?

**Review by rickaw — APPROVED** [review]

File: src/game.css (hunk 1 above)
Comment (caira044) [reply src/game.css:17]: Yes, Dimi and I thought this might look better

File: src/game.js (hunk 2 above)
Comment (caira044) [reply src/game.js:51]: yes!

**Pull request merged**